src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs

```diff
@@ -108,7 +102,46 @@ private async Task<TextChange> GenerateSnippetTextChangeAsync(Document document,
             return root.ReplaceNode(snippetExpressionNode, reformatSnippetNode);
         }
 
-        private static SyntaxNode? GetConsoleExpressionStatement(ISyntaxFactsService syntaxFacts, SyntaxNode root, int position)
+        protected override List<(string, List<TextSpan>)> GetRenameLocationsMap(SyntaxNode node, ISyntaxFacts syntaxFacts, CancellationToken cancellationToken)
+        {
+            var renameLocationsMap = new List<(string, List<TextSpan>)>();
+            var openParenToken = GetOpenParenToken(node, syntaxFacts);
+
+            if (openParenToken is null)
+            {
+                return renameLocationsMap;
+            }
+
+            var list1 = new List<TextSpan>
+            {
+                new TextSpan(openParenToken.Value.Span.End, openParenToken.Value.Span.Length)
```

Comment:
i'm a bit confused by this.  You're renaming the span that starts at the end of hte open paren, but then had length 1?